generators/mail/templates/common/manifest.xml

```diff
@@ -7,6 +7,7 @@
   <DefaultLocale>en-US</DefaultLocale>
   <DisplayName DefaultValue="<%= projectDisplayName %>" />
   <Description DefaultValue="[Outlook Add-in description]"/>
+  <SupportUrl DefaultValue="http://contoso.com/support" />
```

Comment:
<SupportUrl> element is only required if the add-in is built to be submitted to the Office Store. For internal use only, it will not be necessary. If you could add a comment to explain it and include a sample such as <!--<SupportUrl DefaultValue="[Outlook Add-in support url]"/>-->, I'd be happy to merge it to the source code.